schedule/cryptlvm_minimal_x.yaml

```diff
@@ -0,0 +1,60 @@
+---
+name:           cryptlvm_minimal_x
+description:    >
+  Combination of "cryptlvm" and "minimal_x" for sle12.
+  (crypt-)LVM installations can take longer,
+  especially on non-x86_64 architectures.
+vars:
+  ENCRYPT: 1  
+  LVM: 1
+  MAX_JOB_TIME: 14400
+schedule:
+  - installation/bootloader
+  - installation/welcome
+  - installation/accept_license
+  - installation/registration
+  - installation/addon_products_sle
+  - installation/system_role
+  - installation/partitioning
+  - installation/partitioning/encrypt_lvm
+  - installation/partitioning_finish
+  - installation/installer_timezone
+  - installation/user_settings
+  - installation/user_settings_root
+  - installation/resolve_dependency_issues
```

Comment:
At the moment named: `installation_overview_before`

Reply:
i will keep it as i am expecting the other one to be merged before this

Reply:
Already merged :)